src/runtime/storage/store-interface.ts

```diff
@@ -200,6 +194,17 @@ export abstract class ActiveMuxer<T extends CRDTTypeRecord> implements StorageCo
     }
   }
 
+  async idle(): Promise<void> {
```

Comment:
this can also be removed: no need to override a super method with the same implementation

Reply:
Done.